initializr-service/src/main/resources/application.yml

```diff
@@ -1198,6 +1206,41 @@ initializr:
           id: remote-shell
           description: CRaSH shell integration
           versionRange: "[1.0.0.RELEASE,1.5.0.RC1)"
+    - name: Spring Cloud GCP
+      bom: spring-cloud-gcp
+      versionRange: 1.0.0.M2
+      content:
+        - name: Spring Cloud GCP Starter
+          id: spring-cloud-gcp-starter
+          description: Starter for Spring Cloud GCP
+          groupId: org.springframework.cloud
+          artifactId: spring-cloud-gcp-starter
+          starter: true
+          links:
+            - rel: reference
+              href: https://docs.spring.io/spring-cloud-gcp/docs/1.0.0.M2/reference/htmlsingle/
+              description: Reference document
+        - name: Spring Cloud GCP Pub/Sub Starter
```

Comment:
None of the entries have the word "starter" in them. Please align with existing entries. We don't use "Spring Cloud" either. 

I'd name that "GCP Pub/Sub" or "GCP Messaging"